pandas/io/formats/style.py

```diff
@@ -178,6 +178,14 @@ class Styler(StylerRenderer):
     escape : str, optional
         Use 'html' to replace the characters ``&``, ``<``, ``>``, ``'``, and ``"``
         in cell display string with HTML-safe sequences.
+
+        .. warning::
+
+           ``Styler`` is primarily intended for use on safe input that you control.
+           When using ``Styler`` on untrusted, user-provided input to serve HTML,
+           you should set ``escape=true`` to prevent security vulnerabilities.
```

Comment:
true is not an option. For untrusted HTML one should use ``escape='html'``. Also does putting a warning in the middle of the docstring render ok? This separates the HTML part from the LaTex part. If it renders OK im fine with it.

Reply:
Here's how it renders:

<img width="531" alt="image" src="https://github.com/pandas-dev/pandas/assets/1312546/46feb285-51c1-4e9e-84d6-e36b2293d631">


Maybe the `..warning` would be better under "Notes". The `escape` section could say to see the Notes about escaping HTML.

Reply:
Yeh I agree, its not really relevant to the input the argument but more for thedesign or best practice programs. Is there a relevant spot in the notes you can add it to?

Reply:
And don't forget that `True`is not an option.

Reply:
Moved to Notes and fixed the example to say `escape="html"`.

> Is there a relevant spot in the notes you can add it to?

I'm not sure... I didn't see a good section (it's not really a bug fix) and I don't think it's really worth a new section.